js/index.js

```diff
@@ -1,4 +1,21 @@
 var environment = require('./environment')
+var apod = require('nasa-apod')
+var $ = require('jquery')
+
+var randomDate = function randomDate(start, end) {
+    return new Date(start.getTime() + Math.random() * (end.getTime() - start.getTime()));
+}
```

Comment:
feel like this either belongs in `./js/services/random-date.js` or we can use this [already built module](https://www.npmjs.com/package/random-date)